### PR TITLE
Feat: adding naive transaction implementation

### LINKED ISSUE #80: Transactions support
Hey, first of all nice work with this library!

I did a [naive implementation](https://github.com/wovalle/fireorm/blob/master/test/monkey-patch-firestore-transaction.ts) to support transactions and I'm monkey patching your library in my project. It doesn't have any special checks/corner cases, just exposes a compatible api for firestore transactions.

Would you be interested on a PR adding this transactions naive implementation to the library?

Thanks!

### PR DESCRIPTION
**Pull request by wovalle**
Closes #80 

As discussed in the issue, I added a naive implementation of transactions. It doesn't do any check (throwing if you try to call get after calling a set/update/delete) but is a good start :)

### CODE REVIEW

File: src/firebase/firestore/transaction/index.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+  update(ref, data) {
+    return ref.update(data);
+  }
+
+  get(ref) {
+    return ref.get();
+  }
+
+  getAll(...refs) {
```
Comment (mikkopaderes) [inline src/firebase/firestore/transaction/index.js]: I don't see this as a public API [here](https://firebase.google.com/docs/reference/js/firebase.firestore.Transaction). Should we remove this?

Comment (wovalle) [reply src/firebase/firestore/transaction/index.js]: I actually checked [firestore client](https://github.com/googleapis/nodejs-firestore/blob/master/dev/src/transaction.ts#L188-L207) code to copy the API, now I saw that it is private. Will remove.

**Comment by mikkopaderes** [comment]
Thanks for this. Really appreciate it!

**Comment by wovalle** [comment]
No, thanks to you! This library has been super useful to create unit tests in my project. This PR is just a way to say thanks for your good work!

**Comment by wovalle** [comment]
Let me know when the new version is published so I can update and remove the code from my project :D
